samples/Chat/src/app/ChatScreen.tsx

```diff
@@ -58,6 +58,8 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
           endChatHandler(removedBy !== userId);
         }
       });
+      // set topic to be Your Chat sample after getting adapter from createAzureCommunicationChatAdapter
+      // this is executed everytime when any of displayName, endpointUrl, threadId, token, userId, errorHandler, endChatHandler changes
       adapter.setTopic('Your Chat sample');
```

Comment:
My concern (reason for filing the original bug) wasn't so much about this needing a comment, but more that anytime a user joins the chat, they are going to reset the topic. This line changes the topic for everyone. As an example, if Contoso added code to update the topic to something like 'Chat with x,y,x', when a new user joins this will overwrite that set topic back to 'Your Chat sample'

Reply:
Updated my PR

Reply:
Thanks! Update PR title and desc to match also